README.md

```diff
@@ -41,31 +38,28 @@ language that you can read [here](https://rust-unofficial.github.io/patterns/).
 * TODO taking an enum rather than having multiple functions
 * TODO `unwrap()`ing every `Result` instead of forwarding it
 
-
 ## Contributing
 
 You are missing content in this repository that can be helpful for others and you are eager to explain it?
 Awesome! We are always happy about new contributions (e.g. elaboration or corrections on certain topics) to this project.
 
 We suggest reading our [Contribution guide](./CONTRIBUTING.md) to get more information on how it works.
 
-
 ## Building with mdbook
 
 This book is built with [mdbook](https://rust-lang.github.io/mdBook/). You can install it by running `cargo install mdbook`.
 
 If you want to build it locally you can run one of these two commands in the root directory of the repository:
 
-- `mdbook build`
+* `mdbook build`
```

Comment:
Why is it doing this one? Was the one before not valid markdown? 🤔

Reply:
I can ban this rule by adding it into .markdownlint.json but it's MD004, where it prefers asterisks instead of dashes.

Reply:
i tend to use dashes for uls myself as well. let me fix that.

Reply:
k